dev-tools/validation/check-coderabbit-review.py

```
@@ -0,0 +1,270 @@
+        nodes {
+          author { login }
+          body
+          createdAt
+          url
+        }
+      }
+    }
+  }
+}
+""".strip()
+    completed = subprocess.run(
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)